packages/typo3-docs-theme/src/Settings/Typo3DocsThemeSettings.php

```diff
@@ -4,17 +4,31 @@
 
 class Typo3DocsThemeSettings
 {
+    /**
+     * @var array<string, string>
+     */
+    private static array $defaultSettings = [
+        'typo3_core_dev' => "main",
+        'typo3_core_latest' => "12.4",
```

Comment:
Why distinguish between latest and stable?

Reply:
Once 13.0 is released that would be latest but not yet stable?

Reply:
But we don't have 13.0 manuals, we have only main manuals until 13.4 is released. And 13.4 will then become stable.

Reply:
So I removed it